agent/consul/config_replication.go

```diff
@@ -98,12 +101,13 @@ func (s *Server) reconcileLocalConfig(ctx context.Context, configs []structs.Con
 		}
 
 		resp, err := s.raftApply(structs.ConfigEntryRequestType, &req)
-		if err != nil {
-			return false, fmt.Errorf("Failed to apply config %s: %v", op, err)
+		if err == nil {
```

Comment:
This section had to be different since there are TWO error branches on 1.9 instead of just the one.